libsel4vm/src/arch/arm/vgic/vgic_v2.c

```diff
@@ -236,6 +297,9 @@ static vm_frame_t vgic_vcpu_iterator(uintptr_t addr, void *cookie)
  */
 int vm_install_vgic(vm_t *vm)
 {
+    /* Read device tree to find memory regions */
+    gic_get_registers_from_fdt(vm, &dist_addr, &cpu_addr, &vcpu_addr);
```

Comment:
We should probably check and return the error here if the GIC can't be found in the FDT.